src/streamlink/plugins/pandalive.py

```diff
@@ -0,0 +1,74 @@
+import logging
+import re
+
+from streamlink.plugin import Plugin, PluginError, pluginmatcher
+from streamlink.plugin.api import validate
+from streamlink.stream.hls import HLSStream
+
+log = logging.getLogger(__name__)
+
+
+@pluginmatcher(re.compile(
+    r"https?://(?:www\.)?pandalive\.co\.kr/"
+))
+class Pandalive(Plugin):
+    API_URL = "https://api.pandalive.co.kr/v1/live/play"
+
+    _room_id_re = re.compile(r"roomid\s*=\s*String\.fromCharCode\((.*)\)")
+
+    def _get_media_code(self):
+        try:
+            return self.session.http.get(self.url, schema=validate.Schema(
+                validate.parse_html(),
+                validate.xml_xpath_string(".//script[contains(text(), 'roomid')]/text()"),
+                str,
+                validate.transform(self._room_id_re.search),
+                validate.get(1),
+                validate.transform(lambda s: "".join(map(lambda c: chr(int(c)), s.split(",")))),
```

Comment:
Please avoid the TypeError on the `validate.get(1)` call if the regex doesn't match and returns None.

```suggestion
                validate.transform(self._room_id_re.search),
                validate.any(None, validate.all(
                    validate.get(1),
                    validate.transform(lambda s: "".join(map(lambda c: chr(int(c)), s.split(","))))
                )
```